packages/document/builder-doc/docs/zh/config/performance/preconnect.md

```diff
@@ -0,0 +1,22 @@
+- **类型：** `undefined | string | PreconnectOption | Array<PreconnectOption>`
```

Comment:
We can make the type simpler, just support Array<string | PreconnectOption> is enough.